src/resolvers/project.ts

```
@@ -185,17 +184,19 @@ export const resolvers: Resolvers = {
           const newProject = new Project({title, isTestProject});
           const created = await newProject.create(context);
 
-          if (!isNullOrUndefined(created) && !created.hasErrors()) {
-            // Set the current user as an owner on the project
-            const ownerWasSet = await setCurrentUserAsProjectOwner(context, created.id);
-            // Set the current user as the default primary contact
-            const contactWasSet = await ensureDefaultProjectContact(context, created);
+          if (!isNullOrUndefined(created)) {
```

We were essentially ignoring all errors add to the object during create